test/ignores.json

```diff
@@ -9,7 +9,6 @@
   "render-tests/fill-extrusion-pattern/literal": "https://github.com/mapbox/mapbox-gl-js/issues/3327",
   "render-tests/fill-extrusion-pattern/missing": "https://github.com/mapbox/mapbox-gl-js/issues/3327",
   "render-tests/fill-extrusion-pattern/opacity": "https://github.com/mapbox/mapbox-gl-js/issues/3327",
-  "render-tests/fill-outline-color/fill": "https://github.com/mapbox/mapbox-gl-js/issues/4601",
```

Comment:
`fill-outline-color/fill` isn't included in the image updates. Was it passing everywhere already?